### DISCUSSION TITLE
Toggle feature cannot have multiple divs to have different status

### DISCUSSION THREAD

**Discussion by zben** [excid3/tailwindcss-stimulus-components#217] [Q&A]
I have 2 divs that I wanna toggle. Div A starts as hidden and Div B starts as not hidden. When I click button, I want A to show and B to hide. The current toggle feature assumes a global data-toggle-open-value. Is it possible to set data-toggle-open-value per child div? or just look at if the div has hidden class or not and toggle them individually? I think that will be more flexible. 

This will be really helpful if I want to have swapping effect and switch between 2 sets of content.

**Comment by shaneshort** [comment]
This is something that I think was working before the move to using the transition method. I've just now noticed that what I had working is now broken (I was using it to toggle an open icon to be replaced by a closed icon for a mobile menu)

**Comment by matt17r** [comment]
It would be very helpful with a number of the TailwindUI layouts/shells.

I suggested this several months after you asked this question (didn't see your question that time) and Chris said it's not currently supported - https://github.com/excid3/tailwindcss-stimulus-components/discussions/250

I'm trying to figure out a solution again for a new layout and I stumbled across my suggestion again 🙃
